src/jit/graph.c

```diff
@@ -3877,6 +3765,82 @@ static MVMint32 consume_ins(MVMThreadContext *tc, MVMJitGraph *jg,
         jg_append_label(tc, jg, reentry_label);
         break;
     }
+    case MVM_OP_sp_runnativecall_v:
+    case MVM_OP_sp_runnativecall_i: {
```

Comment:
We're not JIT-compiling native calls that return floating point, string, or pointer types yet, iiuc?